tests/js/server/aql/aql-optimizer-rule-move-calculations-down.js

```diff
@@ -371,7 +371,7 @@ function optimizerRuleTestSuite () {
         expected.push("test" + i + "-" + i);
       }
 
-      var query = "FOR i IN 0..100 LET result = (UPDATE {_key: CONCAT('test', TO_STRING(i))} WITH {updated: true} IN " + cn + " RETURN CONCAT(NEW._key, '-', NEW.value)) LIMIT 10 RETURN result[0]";
+      var query = "FOR i IN 0..99 LET result = (UPDATE {_key: CONCAT('test', TO_STRING(i))} WITH {updated: true} IN " + cn + " RETURN CONCAT(NEW._key, '-', NEW.value)) LIMIT 10 RETURN result[0]";
```

Comment:
Because there are only 100 documents in the collection.
i = 0; i< 100; ++i
0..100 is equivalent to i =0; i <= 100; ++I
Before: Limit causes only 10 modifications to be executed.
After drilling done that ALL modifications need to be executed this test fails with Document not Found on  `i == 100`